src/climate_data/generate/scenario_daily.py

```diff
@@ -301,28 +265,30 @@ def generate_scenario_daily(
 
     yve = []
     complete = []
-    for y, v, e in itertools.product(years, variables, experiments):
-        path = cd_data.daily_results_path(scenario=e, variable=v, year=y)
+    for d, y, v, e in itertools.product(draws, years, variables, experiments):
+        path = cd_data.daily_results_path_with_draws(
+            scenario=e, variable=v, year=y, draw=d
+        )
         if not path.exists() or overwrite:
-            yve.append((y, v, e))
+            yve.append((d, y, v, e))
         else:
-            complete.append((y, v, e))
+            complete.append((d, y, v, e))
 
     print(f"{len(complete)} tasks already done. " f"Launching {len(yve)} tasks")
     jobmon.run_parallel(
         runner="cdtask",
         task_name="generate scenario_daily",
         flat_node_args=(
-            ("year", "target-variable", "cmip6-experiment"),
+            ("draw", "year", "target-variable", "cmip6-experiment"),
             yve,
         ),
         task_args={
             "output-dir": output_dir,
         },
         task_resources={
             "queue": queue,
-            "cores": 5,
-            "memory": "120G",
+            "cores": 1,
```

Comment:
Rerunning test with these settings

Reply:
How did this go?

Reply:
about the same times, so multiple cores are not being used, I assume - 

runtime:
 billg@long-slurm-sarchive-p0015:/mnt/team/lsae/pub/climate-data/zzz_logs$ source ~/data/clim/runtimepull.sh
      1 00:00:
     18 00:06:
     58 00:07:
     70 00:08:
     92 00:09:
    134 00:10:
    136 00:11:
     78 00:12:
     28 00:13:
     16 00:14:
      4 00:15:                                                                                           

Mem is comparable to previously